src/components/specific/visa/visa-summary/visa-comments/visa-comment-post/VisaCommentPost.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <div
+    class="visa-comment-post"
+    :class="`visa-comment-post${isAReply ? '__reply' : ''}`"
+    v-click-away="toggleCloseActions"
+  >
+    <div class="visa-comment-post__header">
+      <div class="visa-comment-post__header__left-side">
+        <UserAvatar
+          :user="comment.author"
+          size="27"
+          initialsSize="14"
+          color="silver-light"
+          style="box-shadow: var(--box-shadow)"
+        />
+        <BIMDataTextBox
+          class="visa-comment-post__header__left-side__name"
+          :text="comment.fullName || comment.author.email"
+          maxWidth="30%"
+        />
+        <span class="visa-comment-post__header__left-side__separator">•</span>
+        <span class="visa-comment-post__header__left-side__date">{{
+          formatDateDDMMYYYHHMM(comment.createdAt)
```

Comment:
Here you can use `$d` function from [vue-i18n](https://vue-i18n-next.intlify.dev/api/injection.html#d-value) instead of a custom formatter. This will improve code consistency and handle i18n date formatting for you. ;-)

You can see an example usage of this function [here](https://github.com/bimdata/platform-next/blob/develop/src/components/specific/models/models-manager/models-table/ModelsTable.vue#L27).

Don't hesitate to ask if you need more explanation. :)